test/test_autograd.py

```diff
@@ -35,6 +38,7 @@
                                                   IS_WINDOWS, IS_MACOS, CudaMemoryLeakCheck)
 from torch.autograd import Variable, Function, detect_anomaly, kineto_available
 from torch.autograd.function import InplaceFunction
+from torch.autograd import forward_ad as fwAD
```

Comment:
The fact that you need to alias the API like this seems like a pretty clear indication that the default module naming is too wordy for most people to want to use.